src/ctim/schemas/common.cljc

```diff
@@ -377,8 +377,11 @@
 
 (def-map-type Observable
   (f/required-entries
-   (f/entry :value f/any-str)
-   (f/entry :type v/ObservableTypeIdentifier))
+   (f/entry :value f/any-str
+            :default "1.2.3.4"
+            :description "The value of the observable.")
+   (f/entry :type v/ObservableTypeIdentifier
```

Comment:
As you provided a default value for the observable value that is an IPv4 address perhaps you can also set a default value for the observable type?

```suggestion
   (f/entry :type v/ObservableTypeIdentifier
            :default "ip"
```